schemas/schema-schema.ipldsch

```diff
@@ -135,6 +135,35 @@ type RepresentationKind enum {
 	| "link"
 }
 
+## Any is a union of the basic kind types (minus links). Any is used to
+## specify that an IPLD node may be of any of the basic data model kinds.
+##
+## Links are left out of this union for now, and therefore any element in
+## the schema DSL (`&Any`) that uses it. The placement of links should be
+## intentional when designing schemas.
+##
+type Any union {
+	| TypeBool "bool"
+	| TypeString "string"
+	| TypeBytes "bytes"
+	| TypeInt "int"
+	| TypeFloat "float"
+	| TypeMap "map"
+	| TypeList "list"
+} representation kinded
```

Comment:
Here and below, wee typo: kinded unions need no quotes, because kinds are keywords.  (The quoted strings wouldn't show up anywhere in serial form, either, so quoting them would be confusing.)

Reply:
Also, wrong level of meta.   'Any' contains any *value*, not the description of any *type*.  Thus:

```suggestion
-       | TypeBool "bool"
-       | TypeString "string"
-       | TypeBytes "bytes"
-       | TypeInt "int"
-       | TypeFloat "float"
-       | TypeMap "map"
-       | TypeList "list"
+       | Bool bool
+       | String string
+       | Bytes bytes
+       | Int int
+       | Float float
+       | Map map
+       | List list
```

Reply:
Oh shoot.  There's no great definition of `Map` for this purpose.  It would have to be something like `{String:Any}` but that seems a bit shakey itself for putting into the prelude of known type names.  I'm not sure what to do about this.